model-optimizer/extensions/ops/ONNXResize10.py

```diff
@@ -0,0 +1,50 @@
+"""
+ Copyright (C) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import numpy as np
+
+from mo.graph.graph import Node, Graph
+from mo.ops.op import Op
+
+
+class ONNXResize10Op(Op):
+    op = 'ONNXResize10'
```

Comment:
Can we avoid framework specific operations in ops directory? And have just Interpolate?

Reply:
No, because to replace ONNX Resize-10 we need to know input shapes.